mito-ai/mito_ai/completions/handlers.py

```diff
@@ -220,12 +247,13 @@ async def on_message(self, message: str) -> None: # type: ignore
                         self._llm, 
                         message_history, 
                         message_id,
-                        self.reply
+                        self.reply,
+                        model=model or ""
                     )
                     return
                 else:
                     # Regular non-streaming completion
-                    completion = await get_smart_debug_completion(smart_debug_metadata, self._llm, message_history)
+                    completion = await get_smart_debug_completion(smart_debug_metadata, self._llm, message_history, model=model or "")
```

Comment:
If we just handle setting the model right at the top of this function then we don't need any of these `model=model or ""` either. That's one of the benefits of pushing up our type handling